pkg/lib/aws/consts.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2020 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+var (
+	DashboardMinWidthUnits  = 1
```

Comment:
nit: since these constants are specific to cloudwatch, and only used in this package, I think it would make sense to have them in `cloudwatch.go`, and have them be private (e.g. `_dashboardMinWidthUnits = 1`). Do you think that makes sense?

Reply:
I agree. Making this modification.